.github/workflows/ci-migrations.yml

```diff
@@ -0,0 +1,40 @@
+name: Validate Migration Files
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
```

Comment:
could we run this job only when schema was changed?

Reply:
sure, done

Reply:
What about PRs that change the migrations directory without changing the schema?